plugins/chat/app/services/chat/update_user_last_read.rb

```diff
@@ -4,76 +4,76 @@ module Chat
   # Service responsible for updating the last read message id of a membership.
   #
   # @example
-  #  Chat::UpdateUserLastRead.call(user_id: 1, channel_id: 2, message_id: 3, guardian: guardian)
+  #  Chat::UpdateUserLastRead.call(channel_id: 2, message_id: 3, guardian: guardian)
   #
   class UpdateUserLastRead
-    include Service::Base
+    include ::Service::Base
 
-    # @!method call(user_id:, channel_id:, message_id:, guardian:)
-    #   @param [Integer] user_id
+    # @!method call(channel_id:, message_id:, guardian:)
     #   @param [Integer] channel_id
     #   @param [Integer] message_id
     #   @param [Guardian] guardian
-    #   @return [Service::Base::Context]
+    #   @return [Chat::Service::Base::Context]
```

Comment:
I think this is it now  no?


```suggestion
    #   @return [Service::Base::Context]
```

Reply:
I guess, there are a few places that still refer to that namespace (I guess from the zeitwerk move?) so I will update them all